Cargo.toml

```diff
@@ -64,6 +65,11 @@ features = ["composite"]
 version = "0.9.0"
 features = ["executor", "futures-io"]
 
+[dependencies.smithay-egui]
+git = "https://github.com/dragonnn/smithay-egui"
+commit = "1a8a03304eb8fdd5d770144de59f4ea9c0b94d56"
```

Comment:
Isn't this field called `rev`? Cargo complains that this key does not exist.
[Cargo Book Specifying dependencies from git repositories](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)

Reply:
yes, you are right. No idea from where I got the commit idea